src/data/structure/Tuple2.java

```diff
@@ -0,0 +1,26 @@
+package data.structure;
+
+/**
+ * Supports simple pairs of arbitrary items
+ * @author benjaminposerow
+ *
+ * @param <T>
+ * @param <U>
+ */
+public class Tuple2<T,U> {
```

Comment:
Is there really no built-in structure that can do this? I'm not super excited about having to implement something like this (and its sister class, Tuple3) on our own.